src/ai/backend/manager/models/alembic/versions/9e599b62f6f1_update_kernels_vfolder_mounts_vfid_.py

```diff
@@ -0,0 +1,188 @@
+"""update kernels.vfolder_mounts.vfid structure
+
+Revision ID: 9e599b62f6f1
+Revises: 5fbd368d12a2
+Create Date: 2023-06-22 20:57:16.726624
+
+"""
+from uuid import UUID
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.sql.expression import bindparam
+
+from ai.backend.common.types import VFolderMount
+from ai.backend.manager.models import SessionRow, kernels, vfolders
```

Comment:
It is better to avoid directly import the model definitions in alembic migration but manually define the table schema with only relevant columns.
If the database schema evolves in the future, these imports may not represent the state when this migration script is written.